components/LoginForm.vue

```diff
@@ -23,6 +23,11 @@
           :type="showPassword ? 'text' : 'password'"
           @click:append="() => (showPassword = !showPassword)"
         ></v-text-field>
+        <v-checkbox
```

Comment:
Why not do this checkbox in the login page itself?

The form component is primarily for data validation and there's no data to validate here (either true or false). 

Here it won't do any harm but, as (how I understand) the way you actually implemented it, you are passing the data back to the main login page, which only adds duplication.

I did not take my time with Vuex yet so maybe I'm wrong with my guess. If so, please explain me whats the use for the remember me boolean in login.vue

Reply:
There are two ways to login: either by connecting to a public account without password, or by typing your username and password (the usual I'd say). Those two methods use different login paths but end up in the same `$auth.loginWith` function.

In `LoginForm` there's the logic for the usual login. On submit, the call to `this.loginRequest` is actually a call to the action in `store/user.ts` where there, it simply calls `$auth.loginWith` and handles errors. In the page `login`, the `$auth.loginWith` call is made for users authenticating with a public passwordless account and handles it automatically.

So no, the form validation (doesn't seem to me) to only do data validation as it also triggers the login process. This is the place where your username and password are sent from. And I'm not passing the data to the main login page.

Hope it's clearer, and I do agree it's not the clearest part thing but maybe I could look into it sometimes :)

Reply:
Oh, I understand now.

Also my fault while reviewing in the phone, I thought you were setting the vars in ``data()`` in login.vue instead of doing it to the store.

Reply:
Yeah no problem :) I just plugged my bool where the username and password data were already.